Block/Checkout/Cart/ComponentSwitcherProcessor.php

```diff
@@ -45,7 +45,7 @@ public function __construct(
     public function process($jsLayout) {
         // Store Credit
         $jsLayout['components']['block-totals']['children']['storeCredit']['componentDisabled'] =
-            ! $this->configHelper->getStoreCreditConfig();
+            ! $this->configHelper->useStoreCreditConfig();
```

Comment:
Is $storeId parameter needed here as well?

Reply:
@danac-gs i am planning to create separate PR and ticket for frontend part.

Reply:
@danac-gs I checked. it works now for non multiwebsite